### PR TITLE
[#84] Write tutorial about using co-log with custom monad

### PR DESCRIPTION
**Pull request by chshersh**

### CODE REVIEW

File: co-log/example/2-custom/Custom.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+              richMessageAction)
+import Control.Monad.IO.Class (MonadIO)
+import Control.Monad.Reader (MonadReader, ReaderT (..))
+```
+
+## Application environment
+
+If you have complex Haskell application, then most likely you also have
+non-trivial settings that configure your application environment. The
+environment may store various parameters important for work your application.
```
Comment (vrom911) [inline co-log/example/2-custom/Custom.md]:
```suggestion
environment may store various parameters important for the work of your application.
```

File: co-log/example/2-custom/Custom.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+```haskell
+simpleEnv :: Env App
+simpleEnv = Env
+    { envServerPort = 8081
+    , envLogAction  = richMessageAction
+    }
+```
+
+Then we need to define function that performs actions of type `App`:
```
Comment (vrom911) [inline co-log/example/2-custom/Custom.md]:
```suggestion
Then we need to define a function that performs actions of type `App`:
```

File: co-log/example/2-custom/Custom.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+and run our example.
+
+```haskell
+main :: IO ()
+main = runApp simpleEnv example
+```
+
+And the output will look like this:
+
+![Output example](https://user-images.githubusercontent.com/4276606/57191747-79579200-6f5b-11e9-92ed-e7b728a2b9a3.png)
```
Comment (vrom911) [inline co-log/example/2-custom/Custom.md]: I didn't notice that the time formatting is changed! I think that this is the `chronos` library's formatting?

Comment (chshersh) [reply co-log/example/2-custom/Custom.md]: Oops, yea, that might be the reason. Need to investigate formatting in this library...

**Pull request merged**